src/server/Utils/utils.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+def choose_data(nb):
```

Comment:
You may want to use an array for this. If you store all data sets names in an array, then selecting the nth datasets can be simply done by `ds_array[n]`. This is functionally the same, but will help with testing and will make the code more readable.